<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Saranya" failures="1" tests="7" name="com.runner.RunnerTestNg" time="280.188" errors="0" timestamp="2025-04-17T16:08:56 IST" skipped="0">
  <testcase classname="com.runner.RunnerTestNg" name="runScenario" time="37.763"/>
  <testcase classname="com.runner.RunnerTestNg" name="runScenario" time="36.432"/>
  <testcase classname="com.runner.RunnerTestNg" name="runScenario" time="40.339"/>
  <testcase classname="com.runner.RunnerTestNg" name="runScenario" time="36.276"/>
  <testcase classname="com.runner.RunnerTestNg" name="runScenario" time="36.894"/>
  <testcase classname="com.runner.RunnerTestNg" name="runScenario" time="40.988"/>
  <testcase classname="com.runner.RunnerTestNg" name="runScenario" time="51.496">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.definitions.UserAccountDefinition.user_should_see_success_message(UserAccountDefinition.java:93)
at âœ½.user should see "Your address has been successfully added"(file:///C:/Users/saran/git/SeleniumFinalCucumber/ecommerce-lambdatest-bd/src/test/resources/com/features/UserAccountManagement.feature:24)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.runner.RunnerTestNg -->
